cv_next/components/layout/navbar.tsx

```diff
@@ -97,7 +87,7 @@ export default function Navbar() {
             >
               <div className={`absolute bg-black ${isNavbarMenuOpen ? "block" : "hidden"} w-full h-full top-0 left-0 -z-10 opacity-60 rounded-md md:w-0`}></div>
               <ul className="flex flex-col items-center space-y-4 text-primary md:flex-row md:space-x-6 md:space-y-0">
-                {navLinks.map((link) => (
+                {routes.filter(link => link.UILocation === 'navbar').map((link) => (
```

Comment:
All these 'navbar', 'profile' etc - should be constants (preferably in some Definitions file).